src/main/resources/json-schema/export-job-request.json

```diff
@@ -14,23 +14,59 @@
         "attributes": {
           "type": "object",
           "properties": {
-            "params": {
-              "type": "object",
-              "description": "Search parameters used to define objects to run export job on"
+            "searchParams": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "inputField": {
+                    "type": "string",
+                    "description": "JsonPath (mixed notation) of field used to base annotation reasoning on. Indexes must be wildcards",
+                    "examples": [
+                      "digitalSpecimenWrapper.occurrences[*].location.dwc:country"
+                    ]
+                  },
+                  "inputValue": {
+                    "type": "string",
+                    "description": "Value of the field indicated above",
+                    "examples": [
+                      "United Kingdom"
+                    ]
+                  }
+                },
+                "additionalProperties": false
+              }
+            },
+            "targetType": {
```

Comment:
A type would still be good as the enum can contain different types in it. 
https://json-schema.org/understanding-json-schema/reference/enum